components/image/image.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="A design system for enterprise-level products. Clean HTML with component classes .Customizable and theme able."
+    />
+    <link rel="stylesheet" href="/main.css" />
+    <link rel="icon" type="image/x-icon" href="/assests/favicon.ico" />
+    <title>Astro UI | Images</title>
+  </head>
+  <body>
+    <div class="page__container flex--column">
+      <header class="header flex--row">
+        <a href="/index.html">
+          <div class="header__logo-container flex--row">
+            <img
+              src="/assests/zodiaclogo.png"
+              alt="AtroUI logo"
+              class="logo__img"
+            />
+            <h2 class="header__logo">Astro UI</h2>
+          </div>
+        </a>
+        <nav class="header_nav flex--row">
```

Comment:
i see you are using double underscores in class name but here you have used `header_nav` be consistent with your naming

Reply:
oh i didnt see that thankyou for bringing it up to my notice i have fixed it now you can take a look